Makefile

```diff
@@ -0,0 +1,21 @@
+lint:
+	@echo
+	ruff .
+	@echo
+	black --check --diff --color .
+	@echo
+	pip-audit
+
+format:
+	ruff --silent --exit-zero --fix .
+	black .
+
+precommit:
+	make lint
```

Comment:
Should probably have instructions to install 'make' as well, chocolatey seems easiest

Reply:
I believe it's built-in on MacOS and Linux, not sure about Windows.

Reply:
Windows you have to install it 
installing choco with the commad in README and using 
```
choco install make 
```
would be good enough for instructions